### DISCUSSION TITLE
Building the Directus Suite

### DISCUSSION THREAD

**Discussion by Fusseldieb** [directus/directus#3013] [Archive]
Hello,

I wanted to try the `next` build on my server and couldn't get it to combine api+app (Basically making "the suite"). 
Am I missing some crucial step? I already dig through the docs and couldn't find anything related.
It's not a simple `npm run build`, as it seems. This command just generates a `dist/` with some .js files, but it still isn't the final built version.

Or am I not supposed to build Directus on my own? :P

Thanks in advance!

**Comment by rijkvanzanten** [comment]
Directus is built up as a combination of smaller packages that are importing / referencing each other. If you take a look in the source code, you'll find that `directus` imports `@directus/app` to serve under `/admin`, imports `@directus/schema` for schema inspection, etc.

There's no single command to go from this monorepo source into a single bundle that's ready for deployment on a server. We rely on `npm` for distributing the individual parts, which are installed and imported on the server through `npm`.

The best way to run Directus in a non-dev capacity is by using `npx create-directus-project my-project` or by creating a new folder, adding a package.json, and installing directus by running `npm install directus`

**Reply by Fusseldieb** [reply]
Hmmm, interesting! But wouldn't `npx create-directus-project my-project` or `npm install directus` install the main/stable version (8.8.1)?

**Reply by rijkvanzanten** [reply]
Nope! v8 is built in PHP and has a very different deployment flow. v9 is the first version that's on Node and thus distributed through `npm` 🙂

**Reply by Fusseldieb** [reply]
Oh! I totally didn't notice that - somehow. That's why I'm "suddenly" working with node... I was trying to find the logical connection with PHP. :P

Building a PHP app with Node would be very... unique to say the least haha

Now everything makes sense!
